### ISSUE TITLE
Nourishment seems broken

### ISSUE THREAD

**Issue by MissPotato** [TeamAOF/All-of-Fabric-3#231]
For some reason all nourishment is able to do is apply the debuff for having too much sweets. The positive buffs never happen. https://i.imgur.com/wLrytWf.png

**Comment by Tsudico** [comment]
It appears the configuration for nourish is in the wrong directory. Nourish is looking for config/nourish.json and the updated config for nourish is at config/nourish/nourish.json

Copying the one from the new location over the one at the expected location should allow for buffs to be correctly applied but this resulted in an exception when trying to start the specific save or join a server. The problem is the new nourish.json has an error on line 95 and uses "fruits" for group instead of "fruit":
```diff
        {
            "conditions": [
                {
                    "groups": [
-                        "fruits",
+                        "fruit",
                        "vegetables"
                    ],
                    "above": 0.9
```

I have tested the adjusted nourish.json put in the proper path and it does apply buffs.

**Comment by dandin87** [comment]
Thank you for looking into and fixing this! Really appreciate it. I suppose we haven't had enough complaints on the server except when they make all the groups max including sweets and ask why they are malnourished lol.

**Comment by MissPotato** [comment]
This has been fixed in the 2.5.4 version of All of Fabric 3. Thank you.
